### PR TITLE
main/supervisor: upgrade to 4.0.1

### PR DESCRIPTION
**Pull request by J0WI**
http://supervisord.org/changes.html#id2

### CODE REVIEW

**Comment by TBK** [comment]
Please add a subpackage for openrc.

File: main/supervisor/APKBUILD [hunk 1]
```diff
@@ -1,14 +1,15 @@
 # Contributor: Peter Bui <pnutzh4x0r@gmail.com>
 # Maintainer:
 pkgname=supervisor
-pkgver=3.3.5
+pkgver=4.0.1
 pkgrel=0
 pkgdesc="A system for controlling process state under UNIX"
 url="http://supervisord.org"
 arch="noarch"
 license="BSD"
```
Comment (bratkartoffel) [inline main/supervisor/APKBUILD]: It seems they moved to a "custom" licence:
https://github.com/Supervisor/supervisor/blob/master/COPYRIGHT.txt

File: main/supervisor/APKBUILD [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
-depends="python2 py-meld3 py-setuptools"
+depends="python py-meld3 py-setuptools"
```
Comment (ncopa) [inline main/supervisor/APKBUILD]: It does not support `python3`? I think we should explicit depend on the python version number

File: main/supervisor/APKBUILD (hunk 1 above)
Comment (andypost) [reply main/supervisor/APKBUILD]: No idea how to add custom license(

File: main/supervisor/APKBUILD (hunk 2 above)
Comment (TBK) [reply main/supervisor/APKBUILD]: It does, v4 added py3 support.

Comment (andypost) [reply main/supervisor/APKBUILD]: It supports py3 since 4.0 http://supervisord.org/changes.html#id3 that's is why I update dependency as well

**Comment by andypost** [comment]
btw 4.0.2 update with the same questions https://github.com/alpinelinux/aports/pull/7132

**Comment by tcely** [comment]
Please migrate any changes you want to keep to #7132 then close this.

**Pull request closed without merging**